index.js

```diff
@@ -137,12 +137,12 @@ Emitter.prototype.emit = function(){
     flags: this._flags
   };
   var chn = this.prefix + '#' + packet.nsp + '#';
-  var msg = msgpack.encode([uid, packet, opts]);
+  var msg = notepack.encode([uid, packet, opts]);
 
   // publish
   if (opts.rooms && opts.rooms.length) {
     opts.rooms.forEach(function(room) {
-      var chnRoom = chn + room + '#';
+      var chnRoom = chn + room;
```

Comment:
Could you please update this part, to match https://github.com/socketio/socket.io-redis/blob/5.0.0/index.js#L407 (related: https://github.com/socketio/socket.io-redis/pull/151)?